uaclient/contract.py

```diff
@@ -228,3 +244,41 @@ def redact_sensitive(contract_response):
         else:
             redacted[key] = value
     return redacted
+
+
+def request_contract_updates(cfg, contract_token):
+    """Request contract updates from ua-contracts service.
+
+    @param cfg: Instance of UAConfig for this machine.
+    @param contract_token: Token of the contract we are updating.
+
+    @return: True on success False otherwise.
+    """
+    contract_client = UAContractClient(cfg)
+    if not contract_token:  # Crawl the accounts contract routes to get details
+        machine_token = cfg.machine_token['machineToken']
+        try:
+            contract_client.request_accounts(machine_token=machine_token)
+            contract_token = get_contract_token_for_account(
+                contract_client, cfg.accounts[0]['id'],
+                machine_token=machine_token)
+        except util.UrlError as e:
+            logging.error(str(e))
+            print('Could not update contract info from %s' %
```

Comment:
Should this string be in `ua_status`?

Reply:
moved a couple of strings into ua_status